litmus-portal/frontend/src/components/Sections/ChaosWorkflows/BrowseWorkflow/TableData.tsx

```diff
@@ -94,7 +94,7 @@ const TableData: React.FC<TableDataProps> = ({ data }) => {
             value="Workflow"
             onClick={() =>
               history.push({
-                pathname: '/workflow-underground',
+                pathname: '/workflows/workflow-underground',
```

Comment:
I think we should change the name?

Reply:
can you make it a dynamic route `/workflow/:workflowRunId` @S-ayanide

Reply:
There are some technical difficulties with these that @S-ayanide pointed out to me ...we will take this action item later @gdsoumya

Reply:
This route take the user to which page? the workflow details page right, the one with the graph? Or the template details?

Reply:
Yes the one with the graph, but a route with url parameter is already set up for Template, this the /browse and /template approach but it would need some changes in the Tabs which would ultimately change the way the components like Template or Workflow Tables are Scaffolded